CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - Bounties: Remove payout delay ([polkadot-fellows/runtimes#386](https://github.com/polkadot-fellows/runtimes/pull/386))
+- Polkadot System Chains: Reduce the base transaction fee by half
```

Comment:
Should please follow the standard convention of how we mention them here. Will fix it in my MR now.

Reply:
Good point. Another thing we should add to the CI check.